app/models/host.rb

```diff
@@ -1859,6 +1859,10 @@ def read_only_storages
     storages.where(:host_storages => {:read_only => true})
   end
 
+  def available_storages
+    writable_storages.reject(&:maintenance)
```

Comment:
Hmmm...I can't seem to find it, but I thought we had it for folders.  The idea was a class method lived on the model and was sort of like "things available for provisioning".  We also talked about it taking parameters so that the model itself could own the knowledge for filtering.

So, perhaps a method like `provisionable_storages` is a better name.  I would like to move toward a nicer, common interface, but if we are going to introduce *another* new name, perhaps we should do so in a comprehensive manner, instead of a one-off that we may not get back to.